src/components/LowerLayout/GeneralInfo/GeneralInfo.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import global from '../../../styles.module.css';
+import classes from './GeneralInfo.module.css';
+
+const GeneralInfo = (props) => {
+  return (
+    <div className={global.container}>
+      {props.copy.details.map(copy => {
+        return (
+          <div className={classes.vessel} key={copy.heading}>
+            <h3 className={classes.heading}>{ copy.heading }</h3>
+            <div className={classes.content}>
+            <p>{ copy.firstParagraph }</p>
+            
+            { copy.lists.map(list => {
+              return (
+                <ul key={list.list}>
+                  <li>{ list.list }</li>
+                </ul>
+              );
+            }) }
+            {copy.additionalInfo && 
+              copy.additionalInfo.map(info => {
+              return (
+                <p key={info.info}>
+                  { info.info }
```

Comment:
When you put padding between a variable and curly brackets, it implies that you are doing destructuring. https://github.com/airbnb/javascript#destructuring. For example: `const { var1, var2, var3 } = varsObject`. For the sake of clarity JSX templating should not have spaces on the sides. For example: `{info.info}`.